doc/source/release/1.20.0-notes.rst

```diff
@@ -4,3 +4,14 @@
 NumPy 1.20.0 Release Notes
 ==========================
 
+Negation of user-defined BLAS/LAPACK detection order
+----------------------------------------------------
+`distutils` now allows negation of libraries when determining BLAS/LAPACK
+libraries.
+This may be used to remove an item from the library resolution phase, i.e.
+to disallow NetLIB libraries one could do::
+
+   NPY_BLAS_ORDER='^blas' NPY_LAPACK_ORDER='^lapack' python setup.py build
+
+which will use any of the accelerated libraries instead.
+
```

Comment:
You should not edit this file. Instead, make a new file in `doc/release/upcoming_changes`

Reply:
Ok, once everything else has been discussed, I'll move this?

Reply:
Well, fixed it now.